Dockerfile

```diff
@@ -1,2 +1,35 @@
-# Sử dụng Node.js làm base image
-FROM node:16
\ No newline at end of file
+# Use official Node.js as the base image
+FROM node:18 AS build
```

Comment:
use the alpine version of image for smaller size. Also use latest LTS node version (22)